ofctrl/fgraphFlow.go

```diff
@@ -490,7 +779,55 @@ func (self *Flow) installFlowActions(flowMod *openflow13.FlowMod,
 			addActn = true
 
 			log.Debugf("flow install. Added setUDPDst Action: %+v", setUDPDstAction)
+		case "loadAction":
+			loadAct := flowAction.loadAction
+			ofsBits := loadAct.Range.ToOfsBits()
+			loadOfAction := openflow13.NewNXActionRegLoad(ofsBits, loadAct.Field, loadAct.Value)
+
+			err := actInstr.AddAction(loadOfAction, true)
+			if err != nil {
+				return err
+			}
```

Comment:
It's better follow the style of this function implementation. "AddAction" seems will always return nil.

Reply:
done